src/Sulu/Bundle/CoreBundle/Composer/ScriptHandler.php

```diff
@@ -29,7 +29,7 @@ public static function removeComposerLockFromGitIgnore()
         }
 
         $gitignore = file_get_contents(static::GIT_IGNORE_FILE);
-        $gitignore = str_replace('composer.lock' . PHP_EOL, '', $gitignore);
+        $gitignore = str_replace("composer.lock\n", '', $gitignore);
```

Comment:
we should use a regex here for all three available line breaks: `\n\r`, `\r`, `\n`,